notify/notify.py

```diff
@@ -0,0 +1,104 @@
+import json
+import logging
+import requests
+from os import getenv
+from dotenv import load_dotenv
+from amqp_setup import RABBITMQ_HOSTNAME, RABBITMQ_PORT, \
+    connection, channel, \
+    RABBITMQ_HOSTNAME, RABBITMQ_PORT, EXCHANGE, EXCHANGE_TYPE, QUEUES, SUBJECT, MS_BASE_URL, channel_consume
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+load_dotenv()
```

Comment:
Consider writing a config file that loads all the environment variables instead of putting it in amqp_setup.py and notify.py. Can refer to other ms for a similar config file